tests/har.spec.ts

```diff
@@ -281,3 +281,33 @@ it('should have popup requests', async ({ contextFactory, server }, testInfo) =>
   expect(entries[1].request.url).toBe(server.PREFIX + '/one-style.css');
   expect(entries[1].response.status).toBe(200);
 });
+
+it('should have connection details', async ({ contextFactory, server, browserName, platform }, testInfo) => {
+  const { page, getLog } = await pageWithHar(contextFactory, testInfo);
+  await page.goto(server.EMPTY_PAGE);
+  const log = await getLog();
+  const { serverIPAddress, _serverPort: port, _securityDetails: securityDetails } = log.entries[0];
+  expect(serverIPAddress).toMatch(/^127\.0\.0\.1|\[::1\]/);
+  expect(port).toBe(server.PORT);
+  if (browserName === 'webkit' && platform === 'darwin')
+    expect(securityDetails).toEqual({protocol: 'SSL 2.0'});
+  else
+    expect(securityDetails).toBeUndefined();
+});
+
+it('should have security details', async ({ contextFactory, httpsServer, browserName, platform }, testInfo) => {
+  const { page, getLog } = await pageWithHar(contextFactory, testInfo);
+  await page.goto(httpsServer.EMPTY_PAGE);
+  const log = await getLog();
+  const { serverIPAddress, _serverPort: port, _securityDetails: securityDetails } = log.entries[0];
+  expect(serverIPAddress).toMatch(/^127\.0\.0\.1|\[::1\]/);
+  expect(port).toBe(httpsServer.PORT);
+  if (browserName === 'webkit' && platform === 'linux')
+    expect(securityDetails).toBeUndefined();
```

Comment:
Is there a bug in the WebKit linux network stack?

Reply:
TBD—I haven't yet debugged to figure out if it's intended (and simply a shortcoming of Linux WK/libsoup) or if it's just broken.

Reply:
I'm going to try to fix this in WK implementation. At first glance, it looks like it's available via libsoup, just not fully plumbed. FMI [see this Slack thread](https://playwright.slack.com/archives/C0141EZ6ZNH/p1621909162023100?thread_ts=1621906702.021200&cid=C0141EZ6ZNH).

Reply:
See https://github.com/microsoft/playwright/issues/6759.

We'll address this as a follow up, but I don't think it should block this MR.

Reply:
Instead of baking in the bad behavior here, let's mark this as failing on linux.

```ts
it.fail(browserName === 'webkit' && platform === 'linux');
```